github/github.go

```diff
@@ -0,0 +1,54 @@
+package github
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/dbaltas/ergo/repo"
+	"github.com/google/go-github/github"
+	"golang.org/x/oauth2"
+)
+
+//CreateDraftRelease creates a draft release
+func CreateDraftRelease(ctx context.Context, accessToken string, organization string,
```

Comment:
you should create a struct to contain all arguments

Reply:
Will do so after switching to commands.